changelog/seqera-cloud/v24.2.0_cycle28.mdx

```diff
@@ -4,6 +4,24 @@ date: 2025-01-22
 tags: [seqera cloud]
 ---
 
+### Breaking change
+
+- Restore `errorStrategy` default Nextflow config for AWS and GCP launches. The Nextflow job now correctly terminates after pipeline completion. The impact of this change is that pipelines will experience more job failures. Spot failures will no longer be silent and parameters should be set to account for this.
+
+- For AWS this will manifest as generic failures with exit code 1 and messaging indicating termination of host machine. 
+- Spot retries in AWS need to be actively enabled. Alternatively the generic failure is handled at the Nextflow level. 
+
+**You can revert to the previous setting but this is recommended for short-running tasks only:**
+```
+aws.batch.maxSpotAttempts = 5
+google.batch.maxSpotAttempts = 5
+```
+Set `task.maxRetries` to a non-zero value to restore Platform to prior behaviour.
+
+**How do I retry spot failures while making them visible?**
+
+There is no mechanism in Nextflow to specifically control Spot retries. However you can set `process.maxRetries = 5` to retry all failures (including Spot reclamations) a specified number of times before failure.
```

Comment:
```suggestion

```

Reply:
Sorry, actually I'm not sure of this. I'd thought the disabling of aws.batch.maxSpotAttempts happened earlier, and that the most recent change was for `errorStrategy` and `maxRetries` only, but looking now I could be wrong.